code/modules/clothing/suits/toggles.dm

```diff
@@ -59,7 +58,6 @@
 			else
 				H.equip_to_slot_if_possible(hood,slot_head,0,0,1)
 				suittoggled = 1
-				src.icon_state = "[initial(icon_state)]_t"
```

Comment:
The reason I deleted that is that I'd have to make a new (duplicate) icon for the explorer suit otherwise.